src/JuMP.jl

```diff
@@ -161,6 +161,10 @@ mutable struct Model <: AbstractModel
     # Enable extensions to attach arbitrary information to a JuMP model by
     # using an extension-specific symbol as a key.
     ext::Dict{Symbol,Any}
+    # A model-level option that is used as the default for the set_string_name
+    # keyword to @variable and @constraint. This is a special case because it
```

Comment:
Unclear what "This" is referring to. Special case of what?

Reply:
I removed that part of the sentence. I was aiming to describe that this is the first model-level option we've introduced, but perhaps it's not important.